<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="5" failed="15" total="52" passed="32">
  <reporter-output>
  </reporter-output>
  <suite name="TestRun" duration-ms="422067" started-at="2014-12-22T09:41:36Z" finished-at="2014-12-22T09:48:38Z">
    <groups>
    </groups>
    <test name="KhpppRun" duration-ms="422067" started-at="2014-12-22T09:41:36Z" finished-at="2014-12-22T09:48:38Z">
      <class name="khppp.tests.UsersTest">
        <test-method status="PASS" signature="configure()[pri:0, instance:khppp.tests.UsersTest@4b741d6d]" name="configure" is-config="true" duration-ms="8631" started-at="2014-12-22T11:42:28Z" finished-at="2014-12-22T11:42:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configure -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:khppp.tests.UsersTest@4b741d6d]" name="setUp" is-config="true" duration-ms="972" started-at="2014-12-22T11:42:36Z" finished-at="2014-12-22T11:42:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="addUserCustomEmail(java.util.List)[pri:0, instance:khppp.tests.UsersTest@4b741d6d]" name="addUserCustomEmail" duration-ms="19480" started-at="2014-12-22T11:42:37Z" data-provider="testData" finished-at="2014-12-22T11:42:57Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Admin_Admin, Admin111, Ivan, Ivanov, ivanov@epam.com]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addUserCustomEmail -->
        <test-method status="PASS" signature="adminAddNewUser(java.util.List)[pri:0, instance:khppp.tests.UsersTest@4b741d6d]" name="adminAddNewUser" duration-ms="13141" started-at="2014-12-22T11:42:57Z" data-provider="testData" finished-at="2014-12-22T11:43:10Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Admin_Admin, Admin111, IvanMentor, IvanovMentor, MENTOR]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- adminAddNewUser -->
        <test-method status="PASS" signature="adminAddNewUser(java.util.List)[pri:0, instance:khppp.tests.UsersTest@4b741d6d]" name="adminAddNewUser" duration-ms="13074" started-at="2014-12-22T11:43:10Z" data-provider="testData" finished-at="2014-12-22T11:43:23Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Admin_Admin, Admin111, ILabAuto, LabAuto, LAB_MANAGER]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- adminAddNewUser -->
        <test-method status="PASS" signature="adminAddNewUser(java.util.List)[pri:0, instance:khppp.tests.UsersTest@4b741d6d]" name="adminAddNewUser" duration-ms="12671" started-at="2014-12-22T11:43:23Z" data-provider="testData" finished-at="2014-12-22T11:43:36Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Admin_Admin, Admin111, Imentee, Imentee, MENTEE]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- adminAddNewUser -->
        <test-method status="PASS" signature="adminAddNewUser(java.util.List)[pri:0, instance:khppp.tests.UsersTest@4b741d6d]" name="adminAddNewUser" duration-ms="13550" started-at="2014-12-22T11:43:36Z" data-provider="testData" finished-at="2014-12-22T11:43:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Admin_Admin, Admin111, IAdmin, IAdmin, ADMIN]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- adminAddNewUser -->
        <test-method status="PASS" signature="addUserWithoutName(java.util.List)[pri:0, instance:khppp.tests.UsersTest@4b741d6d]" name="addUserWithoutName" duration-ms="2547" started-at="2014-12-22T11:43:49Z" data-provider="testData" finished-at="2014-12-22T11:43:52Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Admin_Admin, Admin111, Petro, Petrov, You input first name incorrect.]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addUserWithoutName -->
        <test-method status="PASS" signature="addUserWithoutSurname(java.util.List)[pri:0, instance:khppp.tests.UsersTest@4b741d6d]" name="addUserWithoutSurname" duration-ms="2363" started-at="2014-12-22T11:43:52Z" data-provider="testData" finished-at="2014-12-22T11:43:54Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Admin_Admin, Admin111, Petro, Petrov, You input last name incorrect.]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addUserWithoutSurname -->
        <test-method status="PASS" signature="shutDown()[pri:0, instance:khppp.tests.UsersTest@4b741d6d]" name="shutDown" is-config="true" duration-ms="249" started-at="2014-12-22T11:43:54Z" finished-at="2014-12-22T11:43:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- shutDown -->
      </class> <!-- khppp.tests.UsersTest -->
      <class name="khppp.tests.LoginTest">
        <test-method status="PASS" signature="configure()[pri:0, instance:khppp.tests.LoginTest@470734c3]" name="configure" is-config="true" duration-ms="31644" started-at="2014-12-22T11:41:36Z" finished-at="2014-12-22T11:42:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configure -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:khppp.tests.LoginTest@470734c3]" name="setUp" is-config="true" duration-ms="35" started-at="2014-12-22T11:42:08Z" finished-at="2014-12-22T11:42:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="userEmptyLogin(java.util.List)[pri:0, instance:khppp.tests.LoginTest@470734c3]" name="userEmptyLogin" duration-ms="8015" started-at="2014-12-22T11:42:11Z" data-provider="testData" finished-at="2014-12-22T11:42:19Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Ivan, Ivanov, Incorrect login or password]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- userEmptyLogin -->
        <test-method status="PASS" signature="userEmptyPassword(java.util.List)[pri:0, instance:khppp.tests.LoginTest@470734c3]" name="userEmptyPassword" duration-ms="834" started-at="2014-12-22T11:42:20Z" data-provider="testData" finished-at="2014-12-22T11:42:20Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Ivan, Ivanov, Incorrect login or password]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- userEmptyPassword -->
        <test-method status="PASS" signature="userIncorrectLogin(java.util.List)[pri:0, instance:khppp.tests.LoginTest@470734c3]" name="userIncorrectLogin" duration-ms="799" started-at="2014-12-22T11:42:20Z" data-provider="testData" finished-at="2014-12-22T11:42:21Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Admin_Admin, 111.0, Incorrect login or password]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- userIncorrectLogin -->
        <test-method status="PASS" signature="userLogin(java.util.List)[pri:0, instance:khppp.tests.LoginTest@470734c3]" name="userLogin" duration-ms="3394" started-at="2014-12-22T11:42:21Z" data-provider="testData" finished-at="2014-12-22T11:42:25Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Admin_Admin, Admin111, Admin Admin (admin)]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- userLogin -->
        <test-method status="PASS" signature="userLogin(java.util.List)[pri:0, instance:khppp.tests.LoginTest@470734c3]" name="userLogin" duration-ms="1136" started-at="2014-12-22T11:42:25Z" data-provider="testData" finished-at="2014-12-22T11:42:26Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Mentor_Mentor, Mentor111, Mentor Mentor (mentor)]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- userLogin -->
        <test-method status="PASS" signature="userLogin(java.util.List)[pri:0, instance:khppp.tests.LoginTest@470734c3]" name="userLogin" duration-ms="1413" started-at="2014-12-22T11:42:26Z" data-provider="testData" finished-at="2014-12-22T11:42:27Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[LabManager_LabManager, LabManager111, LabManager LabManager (lab manager)]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- userLogin -->
        <test-method status="PASS" signature="shutDown()[pri:0, instance:khppp.tests.LoginTest@470734c3]" name="shutDown" is-config="true" duration-ms="270" started-at="2014-12-22T11:42:27Z" finished-at="2014-12-22T11:42:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- shutDown -->
      </class> <!-- khppp.tests.LoginTest -->
      <class name="khppp.tests.SubgroupTest">
        <test-method status="PASS" signature="configure()[pri:0, instance:khppp.tests.SubgroupTest@2eae8e6e]" name="configure" is-config="true" duration-ms="8202" started-at="2014-12-22T11:45:04Z" finished-at="2014-12-22T11:45:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configure -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:khppp.tests.SubgroupTest@2eae8e6e]" name="setUp" is-config="true" duration-ms="542" started-at="2014-12-22T11:45:12Z" finished-at="2014-12-22T11:45:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="preConditions(java.util.List)[pri:0, instance:khppp.tests.SubgroupTest@2eae8e6e]" name="preConditions" duration-ms="1961" started-at="2014-12-22T11:45:12Z" data-provider="testData" finished-at="2014-12-22T11:45:14Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[LabManager_LabManager, LabManager111]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preConditions -->
        <test-method status="FAIL" signature="addNewEmptySubgroup(java.util.List)[pri:0, instance:khppp.tests.SubgroupTest@2eae8e6e]" name="addNewEmptySubgroup" duration-ms="0" started-at="2014-12-22T11:45:15Z" data-provider="testData" depends-on-methods="khppp.tests.SubgroupTest.preConditions" finished-at="2014-12-22T11:45:15Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:161)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: java.lang.NullPointerException
	at khppp.excel.utils.ExcelReader.getCellData(ExcelReader.java:69)
	at khppp.excel.utils.ExcelReader.getRowData(ExcelReader.java:57)
	at khppp.excel.utils.ExcelReader.getTableArray(ExcelReader.java:45)
	at khppp.core.CoreTest.testData(CoreTest.java:58)
	at khppp.tests.BaseCase.testData(BaseCase.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:135)
	... 23 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addNewEmptySubgroup -->
        <test-method status="FAIL" signature="addNewSubgroup(java.util.List)[pri:0, instance:khppp.tests.SubgroupTest@2eae8e6e]" name="addNewSubgroup" duration-ms="0" started-at="2014-12-22T11:45:15Z" data-provider="testData" depends-on-methods="khppp.tests.SubgroupTest.preConditions" finished-at="2014-12-22T11:45:15Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:161)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: java.lang.NullPointerException
	at khppp.excel.utils.ExcelReader.getCellData(ExcelReader.java:69)
	at khppp.excel.utils.ExcelReader.getRowData(ExcelReader.java:57)
	at khppp.excel.utils.ExcelReader.getTableArray(ExcelReader.java:45)
	at khppp.core.CoreTest.testData(CoreTest.java:58)
	at khppp.tests.BaseCase.testData(BaseCase.java:54)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:135)
	... 23 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addNewSubgroup -->
        <test-method status="PASS" signature="logout()[pri:0, instance:khppp.tests.SubgroupTest@2eae8e6e]" name="logout" is-config="true" duration-ms="289" started-at="2014-12-22T11:45:15Z" finished-at="2014-12-22T11:45:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method status="PASS" signature="shutDown()[pri:0, instance:khppp.tests.SubgroupTest@2eae8e6e]" name="shutDown" is-config="true" duration-ms="139" started-at="2014-12-22T11:45:15Z" finished-at="2014-12-22T11:45:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- shutDown -->
      </class> <!-- khppp.tests.SubgroupTest -->
      <class name="khppp.tests.GroupJournalTests">
        <test-method status="PASS" signature="configure()[pri:0, instance:khppp.tests.GroupJournalTests@d86a6f]" name="configure" is-config="true" duration-ms="10403" started-at="2014-12-22T11:46:18Z" finished-at="2014-12-22T11:46:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configure -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:khppp.tests.GroupJournalTests@d86a6f]" name="setUp" is-config="true" duration-ms="160" started-at="2014-12-22T11:46:28Z" finished-at="2014-12-22T11:46:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="preConditions(java.util.List)[pri:0, instance:khppp.tests.GroupJournalTests@d86a6f]" name="preConditions" duration-ms="1508" started-at="2014-12-22T11:46:29Z" data-provider="testData" finished-at="2014-12-22T11:46:30Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[LabManager_LabManager, LabManager111, , , , , ]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preConditions -->
        <test-method status="FAIL" signature="validWithoutSubgroupTitle()[pri:0, instance:khppp.tests.GroupJournalTests@d86a6f]" name="validWithoutSubgroupTitle" duration-ms="1216" started-at="2014-12-22T11:46:30Z" finished-at="2014-12-22T11:46:31Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: is <true>
     but: was <false>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: is <true>
     but: was <false>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
	at khppp.tests.GroupJournalTests.validWithoutSubgroupTitle(GroupJournalTests.java:117)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validWithoutSubgroupTitle -->
        <test-method status="PASS" signature="workOfBackButton()[pri:0, instance:khppp.tests.GroupJournalTests@d86a6f]" name="workOfBackButton" duration-ms="1305" started-at="2014-12-22T11:46:31Z" finished-at="2014-12-22T11:46:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- workOfBackButton -->
        <test-method status="PASS" signature="creationOfMentee(java.util.List)[pri:0, instance:khppp.tests.GroupJournalTests@d86a6f]" name="creationOfMentee" duration-ms="2434" started-at="2014-12-22T11:46:33Z" data-provider="testData" depends-on-methods="khppp.tests.GroupJournalTests.preConditions" finished-at="2014-12-22T11:46:35Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[LabManager_LabManager, LabManager111, Anton, Antonov, MENTEE, , ]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- creationOfMentee -->
        <test-method status="FAIL" signature="correctAddSubgroupBtnName()[pri:0, instance:khppp.tests.GroupJournalTests@d86a6f]" name="correctAddSubgroupBtnName" duration-ms="32137" started-at="2014-12-22T11:46:35Z" depends-on-methods="khppp.tests.GroupJournalTests.forVerifyGUI" finished-at="2014-12-22T11:47:07Z">
          <exception class="org.openqa.selenium.TimeoutException">
            <message>
              <![CDATA[Timed out after 30 seconds waiting for visibility of element located by By.xpath: //div[@class='submenu']/a[text()='Add Subgroup']
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=dd32a7a8-bfdb-4842-9d30-47ff41b73d7a, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: dd32a7a8-bfdb-4842-9d30-47ff41b73d7a]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.TimeoutException: Timed out after 30 seconds waiting for visibility of element located by By.xpath: //div[@class='submenu']/a[text()='Add Subgroup']
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=dd32a7a8-bfdb-4842-9d30-47ff41b73d7a, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: dd32a7a8-bfdb-4842-9d30-47ff41b73d7a
	at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:79)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:228)
	at khppp.application.components.Component.waitFor(Component.java:73)
	at khppp.application.components.Component.waitFor(Component.java:83)
	at khppp.application.components.GroupJournalPage.isBtnAddSubgroupDisplayed(GroupJournalPage.java:31)
	at khppp.application.steps.GroupJournalSteps.isAddSubgroupDisplayed(GroupJournalSteps.java:45)
	at khppp.tests.GroupJournalTests.correctAddSubgroupBtnName(GroupJournalTests.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":"//div[@class='submenu']/a[text()='Add Subgroup']"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf', time: '2014-10-23 20:02:37'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
Command duration or timeout: 10.37 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=dd32a7a8-bfdb-4842-9d30-47ff41b73d7a, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: dd32a7a8-bfdb-4842-9d30-47ff41b73d7a
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:408)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:204)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:156)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:599)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:352)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:449)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:357)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:344)
	at org.openqa.selenium.support.ui.ExpectedConditions.findElement(ExpectedConditions.java:730)
	at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:39)
	at org.openqa.selenium.support.ui.ExpectedConditions$4.apply(ExpectedConditions.java:130)
	at org.openqa.selenium.support.ui.ExpectedConditions$4.apply(ExpectedConditions.java:126)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:208)
	... 32 more
Caused by: org.openqa.selenium.remote.ScreenshotException: Screen shot has been taken
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:136)
	... 42 more
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":"//div[@class='submenu']/a[text()='Add Subgroup']"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf', time: '2014-10-23 20:02:37'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
	at <anonymous class>.FirefoxDriver.prototype.findElementInternal_(file:///C:/Users/YEHOR_~1/AppData/Local/Temp/anonymous8143357805961477093webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9641:26)
	at <anonymous class>.fxdriver.Timer.prototype.setTimeout/<.notify(file:///C:/Users/YEHOR_~1/AppData/Local/Temp/anonymous8143357805961477093webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:548:5)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.TimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- correctAddSubgroupBtnName -->
        <test-method status="FAIL" signature="correctAssignBtnName()[pri:0, instance:khppp.tests.GroupJournalTests@d86a6f]" name="correctAssignBtnName" duration-ms="32727" started-at="2014-12-22T11:47:07Z" depends-on-methods="khppp.tests.GroupJournalTests.forVerifyGUI" finished-at="2014-12-22T11:47:40Z">
          <exception class="org.openqa.selenium.TimeoutException">
            <message>
              <![CDATA[Timed out after 30 seconds waiting for visibility of element located by By.xpath: //a[@class='btn btn-success'][2]
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=dd32a7a8-bfdb-4842-9d30-47ff41b73d7a, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: dd32a7a8-bfdb-4842-9d30-47ff41b73d7a]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.TimeoutException: Timed out after 30 seconds waiting for visibility of element located by By.xpath: //a[@class='btn btn-success'][2]
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=dd32a7a8-bfdb-4842-9d30-47ff41b73d7a, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: dd32a7a8-bfdb-4842-9d30-47ff41b73d7a
	at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:79)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:228)
	at khppp.application.components.Component.waitFor(Component.java:73)
	at khppp.application.components.Component.waitFor(Component.java:83)
	at khppp.application.components.GroupJournalPage.isBtnAssignDisplayed(GroupJournalPage.java:35)
	at khppp.application.steps.GroupJournalSteps.isAssignDisplayed(GroupJournalSteps.java:49)
	at khppp.tests.GroupJournalTests.correctAssignBtnName(GroupJournalTests.java:91)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":"//a[@class='btn btn-success'][2]"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf', time: '2014-10-23 20:02:37'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
Command duration or timeout: 10.52 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=dd32a7a8-bfdb-4842-9d30-47ff41b73d7a, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: dd32a7a8-bfdb-4842-9d30-47ff41b73d7a
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:408)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:204)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:156)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:599)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:352)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:449)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:357)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:344)
	at org.openqa.selenium.support.ui.ExpectedConditions.findElement(ExpectedConditions.java:730)
	at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:39)
	at org.openqa.selenium.support.ui.ExpectedConditions$4.apply(ExpectedConditions.java:130)
	at org.openqa.selenium.support.ui.ExpectedConditions$4.apply(ExpectedConditions.java:126)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:208)
	... 32 more
Caused by: org.openqa.selenium.remote.ScreenshotException: Screen shot has been taken
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:136)
	... 42 more
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":"//a[@class='btn btn-success'][2]"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf', time: '2014-10-23 20:02:37'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
	at <anonymous class>.FirefoxDriver.prototype.findElementInternal_(file:///C:/Users/YEHOR_~1/AppData/Local/Temp/anonymous8143357805961477093webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9641:26)
	at <anonymous class>.fxdriver.Timer.prototype.setTimeout/<.notify(file:///C:/Users/YEHOR_~1/AppData/Local/Temp/anonymous8143357805961477093webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:548:5)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.TimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- correctAssignBtnName -->
        <test-method status="FAIL" signature="correctBackBtnName()[pri:0, instance:khppp.tests.GroupJournalTests@d86a6f]" name="correctBackBtnName" duration-ms="32341" started-at="2014-12-22T11:47:40Z" depends-on-methods="khppp.tests.GroupJournalTests.forVerifyGUI" finished-at="2014-12-22T11:48:12Z">
          <exception class="org.openqa.selenium.TimeoutException">
            <message>
              <![CDATA[Timed out after 30 seconds waiting for visibility of element located by By.xpath:  .//*[@id='group_journal_submit']/div[2]/a
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=dd32a7a8-bfdb-4842-9d30-47ff41b73d7a, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: dd32a7a8-bfdb-4842-9d30-47ff41b73d7a]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.TimeoutException: Timed out after 30 seconds waiting for visibility of element located by By.xpath:  .//*[@id='group_journal_submit']/div[2]/a
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=dd32a7a8-bfdb-4842-9d30-47ff41b73d7a, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: dd32a7a8-bfdb-4842-9d30-47ff41b73d7a
	at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:79)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:228)
	at khppp.application.components.Component.waitFor(Component.java:73)
	at khppp.application.components.Component.waitFor(Component.java:83)
	at khppp.application.components.GroupJournalPage.isBtnBackDisplayed(GroupJournalPage.java:45)
	at khppp.application.steps.GroupJournalSteps.isBackDisplayed(GroupJournalSteps.java:57)
	at khppp.tests.GroupJournalTests.correctBackBtnName(GroupJournalTests.java:103)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":" .//*[@id='group_journal_submit']/div[2]/a"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf', time: '2014-10-23 20:02:37'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
Command duration or timeout: 10.38 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=dd32a7a8-bfdb-4842-9d30-47ff41b73d7a, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: dd32a7a8-bfdb-4842-9d30-47ff41b73d7a
	at sun.reflect.GeneratedConstructorAccessor63.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:408)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:204)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:156)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:599)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:352)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:449)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:357)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:344)
	at org.openqa.selenium.support.ui.ExpectedConditions.findElement(ExpectedConditions.java:730)
	at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:39)
	at org.openqa.selenium.support.ui.ExpectedConditions$4.apply(ExpectedConditions.java:130)
	at org.openqa.selenium.support.ui.ExpectedConditions$4.apply(ExpectedConditions.java:126)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:208)
	... 32 more
Caused by: org.openqa.selenium.remote.ScreenshotException: Screen shot has been taken
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:136)
	... 42 more
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":" .//*[@id='group_journal_submit']/div[2]/a"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf', time: '2014-10-23 20:02:37'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
	at <anonymous class>.FirefoxDriver.prototype.findElementInternal_(file:///C:/Users/YEHOR_~1/AppData/Local/Temp/anonymous8143357805961477093webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9641:26)
	at <anonymous class>.fxdriver.Timer.prototype.setTimeout/<.notify(file:///C:/Users/YEHOR_~1/AppData/Local/Temp/anonymous8143357805961477093webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:548:5)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.TimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- correctBackBtnName -->
        <test-method status="PASS" signature="correctExportBtnName()[pri:0, instance:khppp.tests.GroupJournalTests@d86a6f]" name="correctExportBtnName" duration-ms="18464" started-at="2014-12-22T11:48:12Z" depends-on-methods="khppp.tests.GroupJournalTests.forVerifyGUI" finished-at="2014-12-22T11:48:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- correctExportBtnName -->
        <test-method status="PASS" signature="correctGroupName()[pri:0, instance:khppp.tests.GroupJournalTests@d86a6f]" name="correctGroupName" duration-ms="1841" started-at="2014-12-22T11:48:31Z" depends-on-methods="khppp.tests.GroupJournalTests.forVerifyGUI" finished-at="2014-12-22T11:48:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- correctGroupName -->
        <test-method status="PASS" signature="correctPageName()[pri:0, instance:khppp.tests.GroupJournalTests@d86a6f]" name="correctPageName" duration-ms="60" started-at="2014-12-22T11:48:33Z" depends-on-methods="khppp.tests.GroupJournalTests.forVerifyGUI" finished-at="2014-12-22T11:48:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- correctPageName -->
        <test-method status="PASS" signature="validGroupJournalTableHead()[pri:0, instance:khppp.tests.GroupJournalTests@d86a6f]" name="validGroupJournalTableHead" duration-ms="146" started-at="2014-12-22T11:48:33Z" depends-on-methods="khppp.tests.GroupJournalTests.forVerifyGUI" finished-at="2014-12-22T11:48:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validGroupJournalTableHead -->
        <test-method status="PASS" signature="availableMentees()[pri:0, instance:khppp.tests.GroupJournalTests@d86a6f]" name="availableMentees" duration-ms="1162" started-at="2014-12-22T11:48:33Z" depends-on-methods="khppp.tests.GroupJournalTests.creationOfMentee" finished-at="2014-12-22T11:48:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- availableMentees -->
        <test-method status="PASS" signature="createGroupWithMentee(java.util.List)[pri:0, instance:khppp.tests.GroupJournalTests@d86a6f]" name="createGroupWithMentee" duration-ms="3022" started-at="2014-12-22T11:48:34Z" data-provider="testData" depends-on-methods="khppp.tests.GroupJournalTests.availableMentees" finished-at="2014-12-22T11:48:37Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Anton Antonov,   ,   ,   ,   , My_New_Group, QA]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createGroupWithMentee -->
        <test-method status="PASS" signature="logout()[pri:0, instance:khppp.tests.GroupJournalTests@d86a6f]" name="logout" is-config="true" duration-ms="263" started-at="2014-12-22T11:48:37Z" finished-at="2014-12-22T11:48:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method status="PASS" signature="shutDown()[pri:0, instance:khppp.tests.GroupJournalTests@d86a6f]" name="shutDown" is-config="true" duration-ms="472" started-at="2014-12-22T11:48:38Z" finished-at="2014-12-22T11:48:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- shutDown -->
      </class> <!-- khppp.tests.GroupJournalTests -->
      <class name="khppp.tests.GroupsTest">
        <test-method status="PASS" signature="configure()[pri:0, instance:khppp.tests.GroupsTest@7d0b7e3c]" name="configure" is-config="true" duration-ms="7579" started-at="2014-12-22T11:43:54Z" finished-at="2014-12-22T11:44:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configure -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:khppp.tests.GroupsTest@7d0b7e3c]" name="setUp" is-config="true" duration-ms="184" started-at="2014-12-22T11:44:02Z" finished-at="2014-12-22T11:44:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="preConditions(java.util.List)[pri:0, instance:khppp.tests.GroupsTest@7d0b7e3c]" name="preConditions" duration-ms="0" started-at="2014-12-22T11:44:02Z" data-provider="testData" finished-at="2014-12-22T11:44:02Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:161)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: java.lang.NullPointerException
	at khppp.excel.utils.ExcelReader.getCellData(ExcelReader.java:69)
	at khppp.excel.utils.ExcelReader.getRowContains(ExcelReader.java:84)
	at khppp.core.CoreTest.testData(CoreTest.java:57)
	at khppp.tests.BaseCase.testData(BaseCase.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:135)
	... 23 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preConditions -->
        <test-method status="FAIL" signature="addGroupBtnName()[pri:0, instance:khppp.tests.GroupsTest@7d0b7e3c]" name="addGroupBtnName" duration-ms="34357" started-at="2014-12-22T11:44:02Z" finished-at="2014-12-22T11:44:37Z">
          <exception class="org.openqa.selenium.TimeoutException">
            <message>
              <![CDATA[Timed out after 30 seconds waiting for visibility of element located by By.xpath: //a[contains(.,'Groups')]
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=35c93c1e-0cbc-4aa6-8453-e639dbabc83e, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 35c93c1e-0cbc-4aa6-8453-e639dbabc83e]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.TimeoutException: Timed out after 30 seconds waiting for visibility of element located by By.xpath: //a[contains(.,'Groups')]
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=35c93c1e-0cbc-4aa6-8453-e639dbabc83e, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 35c93c1e-0cbc-4aa6-8453-e639dbabc83e
	at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:79)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:228)
	at khppp.application.components.Component.waitFor(Component.java:73)
	at khppp.application.components.Component.waitFor(Component.java:83)
	at khppp.application.components.NavBar.getTab(NavBar.java:23)
	at khppp.application.steps.NavBarSteps.navigateToGroups(NavBarSteps.java:29)
	at khppp.tests.BaseCase.goToGroupsTab(BaseCase.java:67)
	at khppp.tests.GroupsTest.addGroupBtnName(GroupsTest.java:59)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":"//a[contains(.,'Groups')]"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf', time: '2014-10-23 20:02:37'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
Command duration or timeout: 10.50 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=35c93c1e-0cbc-4aa6-8453-e639dbabc83e, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 35c93c1e-0cbc-4aa6-8453-e639dbabc83e
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:408)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:204)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:156)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:599)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:352)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:449)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:357)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:344)
	at org.openqa.selenium.support.ui.ExpectedConditions.findElement(ExpectedConditions.java:730)
	at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:39)
	at org.openqa.selenium.support.ui.ExpectedConditions$4.apply(ExpectedConditions.java:130)
	at org.openqa.selenium.support.ui.ExpectedConditions$4.apply(ExpectedConditions.java:126)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:208)
	... 33 more
Caused by: org.openqa.selenium.remote.ScreenshotException: Screen shot has been taken
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:136)
	... 43 more
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":"//a[contains(.,'Groups')]"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf', time: '2014-10-23 20:02:37'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
	at <anonymous class>.FirefoxDriver.prototype.findElementInternal_(file:///C:/Users/YEHOR_~1/AppData/Local/Temp/anonymous9033647809687997824webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9641:26)
	at <anonymous class>.fxdriver.Timer.prototype.setTimeout/<.notify(file:///C:/Users/YEHOR_~1/AppData/Local/Temp/anonymous9033647809687997824webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:548:5)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.TimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addGroupBtnName -->
        <test-method status="PASS" signature="exportBtnName()[pri:0, instance:khppp.tests.GroupsTest@7d0b7e3c]" name="exportBtnName" duration-ms="18891" started-at="2014-12-22T11:44:37Z" finished-at="2014-12-22T11:44:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- exportBtnName -->
        <test-method status="PASS" signature="validGroupsTableHead()[pri:0, instance:khppp.tests.GroupsTest@7d0b7e3c]" name="validGroupsTableHead" duration-ms="610" started-at="2014-12-22T11:44:56Z" finished-at="2014-12-22T11:44:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validGroupsTableHead -->
        <test-method status="PASS" signature="validTabName()[pri:0, instance:khppp.tests.GroupsTest@7d0b7e3c]" name="validTabName" duration-ms="484" started-at="2014-12-22T11:44:56Z" finished-at="2014-12-22T11:44:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validTabName -->
        <test-method status="FAIL" signature="addEmptyGroup(java.util.List)[pri:0, instance:khppp.tests.GroupsTest@7d0b7e3c]" name="addEmptyGroup" duration-ms="0" started-at="2014-12-22T11:44:57Z" data-provider="testData" finished-at="2014-12-22T11:44:57Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:161)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: java.lang.NullPointerException
	at khppp.excel.utils.ExcelReader.getCellData(ExcelReader.java:69)
	at khppp.excel.utils.ExcelReader.getRowContains(ExcelReader.java:84)
	at khppp.core.CoreTest.testData(CoreTest.java:57)
	at khppp.tests.BaseCase.testData(BaseCase.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:135)
	... 23 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addEmptyGroup -->
        <test-method status="FAIL" signature="addGroupWithoutDep(java.util.List)[pri:0, instance:khppp.tests.GroupsTest@7d0b7e3c]" name="addGroupWithoutDep" duration-ms="0" started-at="2014-12-22T11:44:57Z" data-provider="testData" finished-at="2014-12-22T11:44:57Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:161)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: java.lang.NullPointerException
	at khppp.excel.utils.ExcelReader.getCellData(ExcelReader.java:69)
	at khppp.excel.utils.ExcelReader.getRowContains(ExcelReader.java:84)
	at khppp.core.CoreTest.testData(CoreTest.java:57)
	at khppp.tests.BaseCase.testData(BaseCase.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:135)
	... 23 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addGroupWithoutDep -->
        <test-method status="FAIL" signature="addGroupWithoutName(java.util.List)[pri:0, instance:khppp.tests.GroupsTest@7d0b7e3c]" name="addGroupWithoutName" duration-ms="0" started-at="2014-12-22T11:44:57Z" data-provider="testData" finished-at="2014-12-22T11:44:57Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:161)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: java.lang.NullPointerException
	at khppp.excel.utils.ExcelReader.getCellData(ExcelReader.java:69)
	at khppp.excel.utils.ExcelReader.getRowContains(ExcelReader.java:84)
	at khppp.core.CoreTest.testData(CoreTest.java:57)
	at khppp.tests.BaseCase.testData(BaseCase.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:135)
	... 23 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addGroupWithoutName -->
        <test-method status="FAIL" signature="groupNameFieldFromSpecialSymbol(java.util.List)[pri:0, instance:khppp.tests.GroupsTest@7d0b7e3c]" name="groupNameFieldFromSpecialSymbol" duration-ms="0" started-at="2014-12-22T11:44:57Z" data-provider="testData" finished-at="2014-12-22T11:44:57Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:161)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: java.lang.NullPointerException
	at khppp.excel.utils.ExcelReader.getCellData(ExcelReader.java:69)
	at khppp.excel.utils.ExcelReader.getRowContains(ExcelReader.java:84)
	at khppp.core.CoreTest.testData(CoreTest.java:57)
	at khppp.tests.BaseCase.testData(BaseCase.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:135)
	... 23 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- groupNameFieldFromSpecialSymbol -->
        <test-method status="FAIL" signature="groupNameFieldOneSymbol(java.util.List)[pri:0, instance:khppp.tests.GroupsTest@7d0b7e3c]" name="groupNameFieldOneSymbol" duration-ms="0" started-at="2014-12-22T11:44:57Z" data-provider="testData" finished-at="2014-12-22T11:44:57Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:161)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: java.lang.NullPointerException
	at khppp.excel.utils.ExcelReader.getCellData(ExcelReader.java:69)
	at khppp.excel.utils.ExcelReader.getRowContains(ExcelReader.java:84)
	at khppp.core.CoreTest.testData(CoreTest.java:57)
	at khppp.tests.BaseCase.testData(BaseCase.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:135)
	... 23 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- groupNameFieldOneSymbol -->
        <test-method status="PASS" signature="validAddGroupTabName()[pri:0, instance:khppp.tests.GroupsTest@7d0b7e3c]" name="validAddGroupTabName" duration-ms="1371" started-at="2014-12-22T11:44:57Z" finished-at="2014-12-22T11:44:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validAddGroupTabName -->
        <test-method status="PASS" signature="verifyButtonsDisplayed()[pri:0, instance:khppp.tests.GroupsTest@7d0b7e3c]" name="verifyButtonsDisplayed" duration-ms="1209" started-at="2014-12-22T11:44:59Z" finished-at="2014-12-22T11:45:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyButtonsDisplayed -->
        <test-method status="PASS" signature="verifyGroupNameField()[pri:0, instance:khppp.tests.GroupsTest@7d0b7e3c]" name="verifyGroupNameField" duration-ms="1215" started-at="2014-12-22T11:45:00Z" finished-at="2014-12-22T11:45:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyGroupNameField -->
        <test-method status="PASS" signature="verifyLabels()[pri:0, instance:khppp.tests.GroupsTest@7d0b7e3c]" name="verifyLabels" duration-ms="1685" started-at="2014-12-22T11:45:01Z" finished-at="2014-12-22T11:45:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLabels -->
        <test-method status="SKIP" signature="creationOfMentee(java.util.List)[pri:0, instance:khppp.tests.GroupsTest@7d0b7e3c]" name="creationOfMentee" duration-ms="1" started-at="2014-12-22T11:45:03Z" data-provider="testData" depends-on-methods="khppp.tests.GroupsTest.preConditions" finished-at="2014-12-22T11:45:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- creationOfMentee -->
        <test-method status="SKIP" signature="displayGroupWithMentee(java.util.List)[pri:0, instance:khppp.tests.GroupsTest@7d0b7e3c]" name="displayGroupWithMentee" duration-ms="1" started-at="2014-12-22T11:45:03Z" data-provider="testData" depends-on-methods="khppp.tests.GroupsTest.creationOfMentee" finished-at="2014-12-22T11:45:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- displayGroupWithMentee -->
        <test-method status="SKIP" signature="menteeInAvailableMenteesField()[pri:0, instance:khppp.tests.GroupsTest@7d0b7e3c]" name="menteeInAvailableMenteesField" duration-ms="0" started-at="2014-12-22T11:45:03Z" depends-on-methods="khppp.tests.GroupsTest.creationOfMentee" finished-at="2014-12-22T11:45:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- menteeInAvailableMenteesField -->
        <test-method status="SKIP" signature="removeMenteeFromGroup()[pri:0, instance:khppp.tests.GroupsTest@7d0b7e3c]" name="removeMenteeFromGroup" duration-ms="1" started-at="2014-12-22T11:45:03Z" depends-on-methods="khppp.tests.GroupsTest.createGroupWithMentee" finished-at="2014-12-22T11:45:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- removeMenteeFromGroup -->
        <test-method status="SKIP" signature="createGroupWithMentee(java.util.List)[pri:0, instance:khppp.tests.GroupsTest@7d0b7e3c]" name="createGroupWithMentee" duration-ms="0" started-at="2014-12-22T11:45:03Z" data-provider="testData" depends-on-methods="khppp.tests.GroupsTest.menteeInAvailableMenteesField" finished-at="2014-12-22T11:45:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createGroupWithMentee -->
        <test-method status="PASS" signature="logout()[pri:0, instance:khppp.tests.GroupsTest@7d0b7e3c]" name="logout" is-config="true" duration-ms="631" started-at="2014-12-22T11:45:03Z" finished-at="2014-12-22T11:45:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method status="PASS" signature="shutDown()[pri:0, instance:khppp.tests.GroupsTest@7d0b7e3c]" name="shutDown" is-config="true" duration-ms="373" started-at="2014-12-22T11:45:03Z" finished-at="2014-12-22T11:45:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- shutDown -->
      </class> <!-- khppp.tests.GroupsTest -->
      <class name="khppp.tests.StatusTest">
        <test-method status="PASS" signature="configure()[pri:0, instance:khppp.tests.StatusTest@6e9175d8]" name="configure" is-config="true" duration-ms="5626" started-at="2014-12-22T11:45:15Z" finished-at="2014-12-22T11:45:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configure -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:khppp.tests.StatusTest@6e9175d8]" name="setUp" is-config="true" duration-ms="209" started-at="2014-12-22T11:45:21Z" finished-at="2014-12-22T11:45:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="preConditionsMentee(java.util.List)[pri:0, instance:khppp.tests.StatusTest@6e9175d8]" name="preConditionsMentee" duration-ms="1842" started-at="2014-12-22T11:45:21Z" data-provider="testData" finished-at="2014-12-22T11:45:23Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Mentee_Mentee, Mentee111]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preConditionsMentee -->
        <test-method status="PASS" signature="preConditionsMentor(java.util.List)[pri:0, instance:khppp.tests.StatusTest@6e9175d8]" name="preConditionsMentor" duration-ms="22437" started-at="2014-12-22T11:45:23Z" data-provider="testData" finished-at="2014-12-22T11:45:45Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Mentor_Mentor, Mentor111]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preConditionsMentor -->
        <test-method status="FAIL" signature="changeStatusByMentee()[pri:0, instance:khppp.tests.StatusTest@6e9175d8]" name="changeStatusByMentee" duration-ms="32363" started-at="2014-12-22T11:45:45Z" depends-on-methods="khppp.tests.StatusTest.preConditionsMentee" finished-at="2014-12-22T11:46:18Z">
          <exception class="org.openqa.selenium.TimeoutException">
            <message>
              <![CDATA[Timed out after 30 seconds waiting for visibility of element located by By.xpath: .//*[@id='0']/td[1]
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=bf3fe967-a367-4ba0-8363-c25053aef59d, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: bf3fe967-a367-4ba0-8363-c25053aef59d]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.TimeoutException: Timed out after 30 seconds waiting for visibility of element located by By.xpath: .//*[@id='0']/td[1]
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=bf3fe967-a367-4ba0-8363-c25053aef59d, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: bf3fe967-a367-4ba0-8363-c25053aef59d
	at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:79)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:228)
	at khppp.application.components.Component.waitFor(Component.java:73)
	at khppp.application.components.Component.waitFor(Component.java:83)
	at khppp.application.components.PersonalJournalPage.clickTaskName(PersonalJournalPage.java:19)
	at khppp.application.steps.StatusSteps.clickTaskNameStep(StatusSteps.java:21)
	at khppp.tests.StatusTest.changeStatusByMentee(StatusTest.java:64)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":".//*[@id='0']/td[1]"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf', time: '2014-10-23 20:02:37'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
Command duration or timeout: 10.50 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=bf3fe967-a367-4ba0-8363-c25053aef59d, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: bf3fe967-a367-4ba0-8363-c25053aef59d
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:408)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:204)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:156)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:599)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:352)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:449)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:357)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:344)
	at org.openqa.selenium.support.ui.ExpectedConditions.findElement(ExpectedConditions.java:730)
	at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:39)
	at org.openqa.selenium.support.ui.ExpectedConditions$4.apply(ExpectedConditions.java:130)
	at org.openqa.selenium.support.ui.ExpectedConditions$4.apply(ExpectedConditions.java:126)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:208)
	... 32 more
Caused by: org.openqa.selenium.remote.ScreenshotException: Screen shot has been taken
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:136)
	... 42 more
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":".//*[@id='0']/td[1]"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf', time: '2014-10-23 20:02:37'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
	at <anonymous class>.FirefoxDriver.prototype.findElementInternal_(file:///C:/Users/YEHOR_~1/AppData/Local/Temp/anonymous1182964102041632494webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9641:26)
	at <anonymous class>.fxdriver.Timer.prototype.setTimeout/<.notify(file:///C:/Users/YEHOR_~1/AppData/Local/Temp/anonymous1182964102041632494webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:548:5)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.TimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- changeStatusByMentee -->
        <test-method status="FAIL" signature="changeStatusByMentor(java.util.List)[pri:0, instance:khppp.tests.StatusTest@6e9175d8]" name="changeStatusByMentor" duration-ms="0" started-at="2014-12-22T11:46:18Z" data-provider="testData" depends-on-methods="khppp.tests.StatusTest.preConditionsMentor" finished-at="2014-12-22T11:46:18Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:161)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: java.lang.NullPointerException
	at khppp.excel.utils.ExcelReader.getCellData(ExcelReader.java:69)
	at khppp.excel.utils.ExcelReader.getRowData(ExcelReader.java:57)
	at khppp.excel.utils.ExcelReader.getTableArray(ExcelReader.java:45)
	at khppp.core.CoreTest.testData(CoreTest.java:58)
	at khppp.tests.BaseCase.testData(BaseCase.java:54)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:135)
	... 23 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- changeStatusByMentor -->
        <test-method status="PASS" signature="shutDown()[pri:0, instance:khppp.tests.StatusTest@6e9175d8]" name="shutDown" is-config="true" duration-ms="206" started-at="2014-12-22T11:46:18Z" finished-at="2014-12-22T11:46:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- shutDown -->
      </class> <!-- khppp.tests.StatusTest -->
    </test> <!-- KhpppRun -->
  </suite> <!-- TestRun -->
</testng-results>
